_i18n/en.yml

```diff
@@ -378,6 +378,52 @@ faq:
   a13: Monero is not magic. If you use Monero but give your name and address to another party, the other party will not magically forget your name and address. If you give out your secret keys, others will know what you've done. If you get compromised, others will be able to keylog you. If you use a weak password, others will be able to brute force your keys file. If you backup your seed in the cloud, you'll be poorer soon.
   q14: Is Monero 100% anonymous?
   a14: There is no such thing as 100% anonymous. If nothing else, your anonymity set is the set of people using Monero. Some people don't use Monero. Monero may also have bugs. Even if not, ways may exist to infer some information through Monero's privacy layers, either now or later. Attacks only get better. If you wear a seatbelt, you can still die in a car crash. Use common sense, prudence and defense in depth.
+  qwherebuy: Where can I buy Monero for Fiat?
+  awherebuy: |
+      Keep in mind the following table is incomplete, and there may be more exchanges out there than what are listed below.
+
+      <table>
```

Comment:
This section doesn't need to be in the variable. Keep in mind that all these strings would be parsed by weblate, which will pull all the HTML with all those strings and show it to the translators, when the strings that would actually need to be translated are only 3 (1 before the table, 2 after).